client/cli/src/client_proxy.rs

```diff
@@ -148,7 +148,10 @@ impl ClientProxy {
 
         let faucet_server = match faucet_server {
             Some(server) => server,
-            None => host.replace("client", "faucet"),
+            None => host
+                .replace("http://", "")
```

Comment:
why need to do this?

Reply:
Other places in code (e.g. `mint_coins_with_faucet_service()` expects `faucet_server` URL to have no prefix, so I just strip all prefixes out here